src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -202,6 +205,7 @@ const formConfig = {
           title: 'Service Information',
           depends: isVeteranOrServiceMember,
           uiSchema: {
+            'ui:description': PrefillMessage,
```

Comment:
@jholton @lihanli will service information always be prefilled? How about address information? If these are not always prefilled we may want to conditionally display this prefill message. @erikphansen we could add a field 'view:serviceInformationPrefilled' and set it to true if service information, for example, is included in the prefilled information, via a prefill transformer (e.g. [0993](https://github.com/department-of-veterans-affairs/vets-website/blob/8e1427fa92037ebdd57f889373caebcd3eeb4d35/src/applications/edu-benefits/0993/helpers.js#L4-L17)), and then check for that value in determining whether to display the prefill message.

Reply:
If a piece of prefill data is unavailable, it will not be in the prefill `formData` json at all (the key will not be present) that the backend returns.

Reply:
Thanks @jholton, in general is only name and email guaranteed to be prefiled for signed in users? Would address and service information only be prefilled for verified users (is not guaranteed to always be prefilled for signed in users)?

Reply:
I honestly have no idea about the general availability of address and service information.
I'd say operating as if only email and name are guaranteed is a good idea.
@lihanli do you have any knowledge as to the reliability of an address or service information being available for signed-in users?

Reply:
neither of those are guaranteed for verified users

Reply:
Thanks @jholton and @lihanli, we'll ensure that those (address, service information) have been prefilled when displaying the message.

Reply:
@erikphansen I think this (checking that individual pages/fields were actually prefilled) is a new pattern, so feel free to create a new ticket if it's something that we decide we want to implement.

Reply:
Done [here](https://app.zenhub.com/workspace/o/department-of-veterans-affairs/vets.gov-team/issues/12607)